cache-paths.js

```diff
@@ -0,0 +1,35 @@
+const homedir = require('user-home');
+const path = require('path');
+
+const env = process.env;
+
+const macos = function (name) {
+	const library = path.join(homedir, 'Library');
+
+	return path.join(library, 'Caches', name);
+};
+
+const windows = function (name) {
```

Comment:
Instead of taking a `name`, can just have `'js-v8flags'` assigned as a variable at the top of file.